packages/fxa-event-broker/lib/db/firestore.ts

```diff
@@ -22,7 +27,7 @@ class FirestoreDatastore implements Datastore {
   }
 
   public async storeLogin(uid: string, clientId: string) {
-    const document = this.db.doc(`users/${uid}`);
+    const document = this.db.doc(this.prefix + `users/${uid}`);
```

Comment:
would it be cleaner to use `${this.prefix}.....`?

Reply:
Sure, why not. 😄